build.sh

```diff
@@ -1,16 +1,16 @@
 #!/bin/bash
 
 read -r -d '' BLOBS_FILES <<- EOM
-https://s3-us-west-2.amazonaws.com/wavefront-cdn/pcf/bosh-artifacts/commons-daemon.tar
-https://s3-us-west-2.amazonaws.com/wavefront-cdn/pcf/bosh-artifacts/zulu8.38.0.13-ca-jdk8.0.212-linux_x64.tar.gz
+http://apache.uvigo.es//commons/daemon/binaries/commons-daemon-1.2.3-bin.tar.gz
```

Comment:
We should host these on s3, so we don't rely on external dependencies (that may change).